apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -143,6 +160,40 @@ class ManageStudentsTable extends Component {
     }
   };
 
+  renderTransferSuccessNotification = () => {
+    const {type, numStudents, sectionDisplay} = this.props.transferStatus;
+    let notification = {};
+
+    switch (type) {
+      case TransferType.MOVE_STUDENTS:
+        notification.notice = i18n.studentsSuccessfullyMovedNotice;
+        notification.details = i18n.studentsSuccessfullyMovedDetails;
+        break;
+      case TransferType.COPY_STUDENTS:
+        notification.notice = i18n.studentsSuccessfullyCopiedNotice;
+        notification.details = i18n.studentsSuccessfullyCopiedDetails;
+        break;
+    }
+
+    return (
+      <Notification
+        type={NotificationType.success}
+        notice={notification.notice()}
+        details={notification.details({numStudents: numStudents, section: sectionDisplay})}
+        dismissible={false}
+      />
+    );
+  };
+
+  studentDataMinusBlanks = () => {
+    return this.props.studentData.filter(sd => sd.rowType === RowType.STUDENT);
+  };
+
+  isMoveStudentsEnabled = () => {
+    const {loginType} = this.props;
+    return loginType === SectionLoginType.word || loginType === SectionLoginType.picture || loginType === SectionLoginType.email;
+  };
```

Comment:
Sorry, one more thing: Can we add a test to [`ManageStudentsTableTest`](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/templates/manageStudents/ManageStudentsTableTest.js) asserting this behavior? (Or maybe its inverse - we don't show a move students button for Google or Clever sections.)